CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Contributor Code of Conduct
```

Comment:
There is a Google modified version of the Code of Conduct available here:

https://github.com/google/new-project/blob/master/docs/code-of-conduct.md

I think we should use that content instead.